### DISCUSSION TITLE
Nuclei False Positive

### DISCUSSION THREAD

**Discussion by omarhashem123** [projectdiscovery/nuclei-templates#5619] [Q&A]
Does anybody know why nuclei when scanning a large number of subdomains give me some false positives but after trying to recheck the vulnerable subdomain only with the same template it scans it without false positives that's means that the problem is not in templates but in nuclei itself
BTW: that happened to me with some templates like put "host-header-injection.yaml", takeovers templates, and other private templates

this issue happened to me while testing a bunch of subdomains in different VPS and in my personal PC and happened to multiple of my friends.

**Comment by forgedhallpass** [comment]
First of all, please don't tag people.

Secondly, the information you've shared is not enough to advise or replicate the above mentioned. If you can reliable reproduce it, please create an issue using the issue template and add all the necessary information like: used command with all the flags, some example input (this you can even share on Discord privatly), the desired behavior and the actual output. Adding screenshots is also preferred.
